configurations/pipeline_config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This should be in a different PR really - it's not adjusting the code schemes as the PR suggests it should.

Reply:
Thank you this was on a previous pending review.